docs/v1.0/api-plugin-output.txt

```diff
@@ -415,11 +415,57 @@ This method is available only for standard format buffer chunks, and to provide
 * ``time``: a Fluent::EventTime object, or an Integer which represents unix timestamp (seconds from Epoch)
 * ``record``: a Hash with String keys
 
-## Writing Tests
+## How to Write Tests
 
-Fluentd output plugin has just one or some points to be tested. Others (parsing configurations, controlling buffers, retries, flushes and many others) are controlled by Fluentd core.
+This section explains how to write a test suite for your custom output plugin.
 
-Fluentd also provides test driver for plugins. You can write tests of your own plugins very easily:
+### Plugin Testing Overview
+
+To write a test suite, you need to define a class inheriting `Test::Unit::TestCase`. The basic structure is mostly same as any other Test::Unit-based test codes. Here is a minimum example.
+
+    :::ruby
+    require_relative '../helper'
+    require 'fluent/test/driver/output'
+    require 'fluent/plugin/out_stdout'
+
+    class MyOutputTest < Test::Unit::TestCase
+        def setup
```

Comment:
nit: We should use 2-spaces indent in code examples.

Reply:
Fixed by 52f4fea.